### DISCUSSION TITLE
Human needs feet?

### DISCUSSION THREAD

**Discussion by joaoburatto** [RE-SS3D/SS3D#716] [Game Design]
Simple question, yes or no and give us your points. 

((This discussion needs to be updated with pictures to help put the discussion into perspective))

**Comment by cosmiccoincidence** [comment]
### YES

- **Artstyle/Realism:**
Although SS13 (and SS3D) is a goofy, quirky, cartoony game... it is also a detailed, complex, simulator that *attempts* to be fairly realistic. Why I bring this up is because there is a fine balance between making an art style that fits both of these attributes of the game, and I think the lack of real feet on the human is an example of this balance leaning too far toward the goofy-cartoon side of the spectrum. The current 'feet' do not resemble feet in the slightest, they look more like horse hooves, which results in the footwear also looking funky. The shoes don't look that terrible but taller footwear really highlights the absurdity. And although I understand the art style, to me the lack of foot is also amplified by the human's giant hands/forearms.

- **Identification/Clarity:**
When I first joined RE:SS3D I remember asking within the first couple of days, "Where are the character's feet?" to be surprised to find out what I thought was just part of the ankle, was in fact what was being considered as a 'foot'. I'm not the only person who has made such a mistake either. If such an identification mistake can be made while the foot is attached to the rest of the human, imagine the difficulty of identifying it if it were severed from the rest of the body.... sure there is a cut limb texture on one end but besides that, it mostly just looks like a cylinder. And yes the player can use the examine function to see when the object is, but the player shouldn't need to rely on this to tell what most things are, it goes without argument that objects should look like the object it is. Another situation of clear identification came when we were making textures for the footprints and were considering using the same shape as the foot itself which was basically a rough circle... we concluded this was too unrecognizable and went with a more elongated rounded foot shape.

- **Consistency:**
It seems that even with the human having essentially no feet, we still plan to make the clown shoes traditionally large, gave the reptilian race feet, and gave other animals feet... but not humans for some reason (I'm not aware of any other entity that doesn't have recognizable feet). This to me doesn't look consistent and also implies a lack of long-term considerations of the artstyle. I think a station full of various player species and animals all running around with normal feet *except for the human* is going to look terribly ugly. And I'm not saying different player species cant have different sized or shaped feet, but we all know the current human foot resembles nothing like a foot and I think it should.

I'll add media to back up my points later on if needed.

**Reply by Kollator** [reply]
IMO the points of consistency and identification would benefit big time from such a change;

- Even when the footprint was deemed to be necessary to be more elongated in shape rather than what the actual foot is right now, why not as well change the actual feet to match more of a foot shape of a human as well?
- Even though lizardmen and such races are based on fantasy, I reckon if they are to have bigger feet as well, the human model would possible then look weird in comparison.
- Identifying the feet when they are detached from the body via mutilation etc. would make the feet more even more identifiable, no?

== I'd give a **yes** to this conversation as well.

**Comment by Dyhr** [comment]
I don't necessarily agree with a no, but I'm gonna give you a no argument, just to play devil's advocate:

> We should stick with the style that was given to us. We did not start this project and so we should not necessarily be changed previous decisions, especially when it's just about the shoes looking a bit silly and it doesn't affect the game that much. The project name is still RE:SS3D and changing the main player model, the model that for sure is the thing players will see the most, would be disrespectful to the original project and the name we took from it.

**Comment by EikoBiko** [comment]
No because:

1. Changing the human model yet again is a pain in the ass -- doing so for severed versions, complete body, and weight transfer model is just unnecessary work nobody wants to do.
2. Changing the shoes we have is unnecessary work nobody wants to do.
3. Stylistically, it doesn't work. I have yet to see anybody propose a solution that people actually liked since it's akin to slapping feet onto the powerpuff girls. If someone can do it, I may change my tune, but the concept of putting significant effort into it doesn't really interest me at all.
> Identifying the feet when they are detached from the body via mutilation etc. would make the feet more even more identifiable, no?
4. There is minimal clarity to be gained from adding feet to models, even when severed. Unless the intent is to add toes (god no), it would still be a challenge to differentiate a severed left foot from right foot. People will almost certainly be examining items to figure these things out regardless.
>Even though lizardmen and such races are based on fantasy, I reckon if they are to have bigger feet as well, the human model would possible then look weird in comparison.
5. Races with digitigrade legs by nature need to include feet in their design, given that the foot is what causes the leg shape to be the way it is. You can integrate the shape of the foot into a digitigrade leg, unlike in the human model, which is stocky and doesn't really lend itself to having feet or toes.
>Even when the footprint was deemed to be necessary to be more elongated in shape rather than what the actual foot is right now, why not as well change the actual feet to match more of a foot shape of a human as well?
6. Better for consistency, sure, but doesn't warrant the proposed change. The purpose of the foot on the human model is different than the purpose of footprints. A footprint looks like a footprint because it needs to be recognizable and convey direction without the context of a body. Feet on the human model need to do nothing but match the rest of the body's style. 
7. Shoes, designed for humans, would look odd on races with significantly thinner frames, like moth people. The silhouette of human-sized feet on moths, which don't even have real feet, would just press the style consistency issue further.

**Reply by Ryan089** [reply]
I'm inclined to agree with Eiko. I think the model looks good as it is. It may not be perfectly realistic, but it is a stylized model that does seem to fit well with the game. Although I'm open to accepting another solution if a volunteer were to offer one, I don't think it's something we should have as a formal task because our current model is perfectly acceptable.

**Reply by Birlinha** [reply]
the base model is great as it is

Is there a way we can make it better? ~(I WANT TOE TRACKING PLEASE)~

For example - Could we "compartmentalize" parts of the 3D model for fracturing later? How will we simulate the effect of cuts, explosions, fractures, etc.?
I believe compartmentalizing the model might be a good idea for computing the fracture easily, but if we could use algorithms to cut holes through our model, for example, I believe would make the game more immersive

**Comment by Birlinha** [comment]
**NO**

I would also argue that

 - Addind toes *might* increase the amount of vertices the models has, increasing in computing complexity, making it harder for computers to simulate the model
 - The increase in Realism and Immersion is not significant ~unless you are a podolatrician~. Seriously, Human is the **only** race that is available right now. 'Plasma Man' are an option, but there is no definitive difference between the 2 main species ~(yes I'm a furry, hm'k?)~
 - With Realism an Immersion in mind, I would argue it would be actually a better idea to work on internal organs. You see - SS13 2D has organ explosions (when you explode someone, their guts fly everywhere) - And it also increases Realism and Immersion for Head Surgeons - Would that be a good idea to add these instead?

**Reply by Ryan089** [reply]
G'day Birlinha, welcome aboard. Thanks for your input. Regarding the organs point, I believe we already have some done, but we haven't yet implemented the code for the health system. [See this image.](https://ss3d.space/assets/img/posts/19.07.01/EikoOrgans.png)
